docs/diagnostics/PX1060.md

```diff
@@ -0,0 +1,68 @@
+# PX1060
+This document describes the PX1060 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                     | Type    | Code Fix  | 
+| ------ | --------------------------------------------------------------------- | ------- | --------- | 
+| PX1060 | DAC fields should be strongly typed to be used in fluent BQL queries. | Message | Available | 
```

Comment:
"fluent BQL" is a technology name so it is always referred to as "Fluent BQL" (with capital "F"), but I'm not sure.

Reply:
we write "fluent BQL" and "traditional BQL" in the documentation